apps/src/p5lab/spritelab/CoreLibrary.js

```diff
@@ -53,6 +57,9 @@ export default class CoreLibrary {
         if (this.screenText.title || this.screenText.subtitle) {
           commands.drawTitle.apply(this);
         }
+        if (this.heading || this.subHeading) {
```

Comment:
For my understanding, what is the distinction between drawing a header and a title (done above)? Maybe a comment for future reference?